app/models/admin_user.rb

```diff
@@ -6,6 +6,5 @@ class AdminUser < ActiveRecord::Base
          :recoverable, :rememberable, :trackable, :validatable
 
   # Setup accessible (or protected) attributes for your model
-  attr_accessible :email, :password, :password_confirmation, :remember_me
```

Comment:
this isn't being called via a controller though so it's not exposed to receive arbitrary params from the world